easybuild/tools/package/utilities.py

```diff
@@ -102,14 +104,20 @@ def package_with_fpm(easyblock):
 
     deps.extend(easyblock.cfg.dependencies())
 
-    _log.debug("The dependencies to be added to the package are: %s",
+    _LOG.debug("The dependencies to be added to the package are: %s",
                pprint.pformat([easyblock.toolchain.as_dict()] + easyblock.cfg.dependencies()))
     depstring = ''
     for dep in deps:
-        _log.debug("The dep added looks like %s ", dep)
+        _LOG.debug("The dep added looks like %s ", dep)
         dep_pkgname = package_naming_scheme.name(dep)
         depstring += " --depends %s" % quote_str(dep_pkgname)
 
+    exclude_files_glob = [
+        os.path.join(log_path(), "*.log"),
+        os.path.join(log_path(), "*.md"),
```

Comment:
add comments here to make it clear that these are (EasyBuild) logs and test report